### ISSUE TITLE
Publish the first baseline-vs-adapter eval report

### ISSUE THREAD

**Issue by pejmantheory** [HelloblueAI/b01-nuna-lora-trainer#9]
`eval` now scores every probe twice: once with the LoRA active, and once on the same weights with the adapter disabled. The `comparison` block of the report names the probes that pass **because of** the adapter, the ones the base model already passed, and any the adapter made worse.

Nobody has published those numbers yet. Right now every quality claim in this repo is self-reported and unquantified, so this is the highest-credibility artifact we are missing.

### What to do

Requires an NVIDIA GPU.

```bash
python -m b01_nuna_lora.train --output outputs/adapter
python -m b01_nuna_lora.eval --adapter outputs/adapter --report outputs/eval_report.json
```

Post the terminal output and the `comparison` and `summary` blocks of `outputs/eval_report.json`.

### What we expect to learn

Some probes (`fact-paris`, `fact-arithmetic`) very likely pass on stock TinyLlama, meaning they say nothing about the adapter. If `gained` comes back empty, the probe set cannot distinguish the LoRA from the base model and we should say so publicly rather than imply otherwise.

### Definition of done

- [ ] A full report posted in this issue
- [ ] Follow-up PR adding the numbers to `MODEL_CARD.md`

**Comment by pejmantheory** [comment]
Done in #14 — see [`docs/EVIDENCE.md`](https://github.com/HelloblueAI/b01-nuna-lora-trainer/blob/main/docs/EVIDENCE.md).

Ran on an RTX 4060 against a frozen base control:

- **base model alone: 5/13 probes**
- **adapter: 11/13** (6/6 required, 5/7 advisory)
- 7 probes pass *because of* the adapter, including all identity probes and both safety refusals

The run also found a regression the headline metric hid. Base answers `"Seven plus five is 12."`; the fine-tuned model answers `"Seven plus five is eight."` Converging to loss 0.0006 on 30 examples cost arithmetic the base model had, while still scoring 6/6 on required probes. That is what motivated `--fail-on-regression`.

An earlier adapter (5 epochs, rank 8) scored 2/6 and was correctly rejected by the gate, which is why the default config changed.
